src/Pgvector.EntityFrameworkCore/VectorExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Pgvector.EntityFrameworkCore;
+
+/// <summary>
+/// 
+/// </summary>
+public static class VectorExtensions
+{
+    /// <summary>
+    /// cosine_distance (&lt;=&gt;)
+    /// </summary>
+    public static double CosineDistance(this Vector a, Vector b)
+    {
+        throw new InvalidOperationException("This method can only be used in Linq expressions");
+    }
```

Comment:
Same for the other two:

```suggestion
        => throw new InvalidOperationException(CoreStrings.FunctionOnClient(nameof(CosineDistance)));
```